appinventor/appengine/tests/com/google/appinventor/server/project/youngandroid/YoungAndroidProjectServiceTest.java

```diff
@@ -0,0 +1,19 @@
+// -*- mode: java; c-basic-offset: 2; -*-
+// Copyright 2009-2011 Google, All Rights reserved
+// Copyright 2011-2012 MIT, All rights reserved
+// Released under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+package com.google.appinventor.server.project.youngandroid;
+
+import junit.framework.TestCase;
+import org.junit.Test;
+
+/* Testing the case of a null build URL */
```

Comment:
I would move this comment ahead of the actual test function. That way if we expand our test coverage later it continues to make sense.